File: Python/EstruturaDeRepeticao/exercicio_14.py
"""
Faça um programa que peça 10 números inteiros, calcule e mostre a quantidade de números pares e a quantidade de números impares.
"""

qtd_pares = 0
qtd_impares = 0

for i in range(1, 11):
    numero = int(input(f"Digite o {i}º número: "))
    if numero % 2 == 0:
        qtd_pares += 1
    else:
        qtd_impares += 1

print(f"Quantidade de números pares: {qtd_pares}")
print(f"Quantidade de números ímpares: {qtd_impares}")

File: Java/EstruturaDeRepeticao/Exercicio14.java
// Faça um programa que peça 10 números inteiros, calcule e mostre a quantidade de números pares e a quantidade de números impares.
package EstruturaDeRepeticao;
import java.util.Scanner;

public class Exercicio14 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int quantidadePares = 0;
        int quantidadeImpares = 0;

        for (int i = 1; i <= 10; i++) {
            System.out.print("Digite o " + i + "º número inteiro: ");
            int numero = scanner.nextInt();

            if (numero % 2 == 0) {
                quantidadePares++;
            } else {
                quantidadeImpares++;
            }
        }

        System.out.println("Quantidade de números pares: " + quantidadePares);
        System.out.println("Quantidade de números ímpares: " + quantidadeImpares);

        scanner.close();
    }
}
